src/faq

```diff
@@ -85,41 +84,14 @@
 
   <h4 id="how-does-the-whatwg-work">How does the WHATWG work?<a class="self-link" href="#how-does-the-whatwg-work"></a></h4>
 
-  <p>People <a href="https://github.com/whatwg">collaborate on GitHub</a> or send email to
-  <a href="/mailing-list#specs">the mailing list</a>.
-
-  <p>Each standard has one or more editors, who are responsible for dealing with feedback for that
```

Comment:
I think some useful content was lost here, especially with the more storytelling style instead of the list-of-requirements style of the working mode document. It might be better to just remove it as it's gotten outdated, but maybe something like this could be added after the working mode:

> In brief, each standard has one or more [editors](https://whatwg.org/working-mode#editor), who guide its evolution. Like a software project, a WHATWG standard undergoes [continual maintenance](https://whatwg.org/faq#living-standard), and occasionally gets [new features](https://whatwg.org/working-mode#additions). This work is driven by the community and in collaboration with implementers.

I'm OK leaving it off though, or modifying the above.